src/pyuvdata/utils/bls.py

```diff
@@ -5,15 +5,59 @@
 import re
 import warnings
 
+import numba
 import numpy as np
+import numpy.typing as npt
 
-from . import _bls
 from .pol import conj_pol, polnum2str, polstr2num
 from .tools import _strict_raise
 
 __all__ = ["baseline_to_antnums", "antnums_to_baseline"]
 
 
+@numba.njit()
+def _max_ant(ant1: npt.NDArray[np.int64], ant2: npt.NDArray[np.int64]) -> np.int64:
+    return np.maximum(np.max(ant1), np.max(ant2))
+
+
+@numba.njit()
+def _min_ant(ant1: npt.NDArray[np.int64], ant2: npt.NDArray[np.int64]) -> np.int64:
+    return np.minimum(np.min(ant1), np.min(ant2))
```

Comment:
@calling `np.ufunc` should be equivalent to `array.ufunc` right?

Reply:
It _should_ be - curious to see if it makes any difference w.r.t. speed (it shouldn't, but doesn't hurt to measure)

Reply:
Yup, no difference

Reply:
thanks for checking at least. Glad they use the same backend